CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Breaking Changes
 
+- Moves `SentryEventDecoder` to SPI (#6365)
+- Adds `final` to more public API (#6365)
```

Comment:
`m`: I think it would be good to exactly point out which API, so everybody knows.